.gitignore

```diff
@@ -100,6 +100,7 @@ ipython_config.py
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
 poetry.lock
+!libs/ragulate/poetry.lock
```

Comment:
why ignore this? I've specifically added it to the repo to control versions of things.

Reply:
eh, this is to not ignore... i get it.